src/pages/guides/index.md

```diff
@@ -256,8 +256,8 @@ For a more robust and reliable verification, Adobe I/O Events adds below securit
 
 **How it strengthens security**
 
-- Adobe I/O Events uses the same public/private key pair generated by itself to sign all event payloads flowing through it's pipeline. The public keys are also hosted at it's own domain i.e. [static.adobeioevents.com](https://static.adobeioevents.com). This ensures that any malicious user can't use his own arbitrary public/private key pair to sign an event payload and send forged requests to your webhook endpoint.
-- Adobe I/O Events sends the event payload with an additional field of `recipient_client_id`. The webhook consumer can match the payload's client id with his own client id to verify if he is the actual recipient of the event. This ensures he doesn't receive events he hasn't subscribed for.
+- Adobe I/O Events uses the same public/private key pair generated by itself to sign all event payloads flowing through it's pipeline. The public keys are also hosted at it's own domain i.e. [static.adobeioevents.com](https://static.adobeioevents.com). This ensures that any malicious user can't use their own arbitrary public/private key pair to sign an event payload and send forged requests to your webhook endpoint.
```

Comment:
We need to replace `it's` with `its`
`The public keys are also hosted at it's own` -> `The public keys are also hosted at its own`